From 09532714c0a8abc2f56003a35775d795349800eb Mon Sep 17 00:00:00 2001
From: Manuel Huber <Manuel.h87@gmail.com>
Date: Mon, 29 Aug 2016 23:19:43 +0200
Subject: [PATCH 01/10] Fix Kernel source path for Yocto

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 5579da6..506e376 100644
--- a/Makefile
+++ b/Makefile
@@ -768,7 +768,7 @@ SUBARCH := $(shell uname -m | sed -e s/i.86/i386/)
 ARCH ?= $(SUBARCH)
 CROSS_COMPILE ?=
 KVER  := $(shell uname -r)
-KSRC := /lib/modules/$(KVER)/build
+KSRC := $(KERNEL_SRC)
 MODDESTDIR := /lib/modules/$(KVER)/kernel/drivers/net/wireless/
 INSTALL_PREFIX :=
 endif
-- 
1.9.1

